README.md

```diff
@@ -1,196 +1,128 @@
+- [Git for Windows](https://gitforwindows.org/)
+
+### Recommended software
+- [epydoc](http://epydoc.sourceforge.net/) - enable PyCharm to parse Core's documentation. Use `py -m pip install epydoc` with your selected python interpreter or install it from PyCharm.
+- [PyCharm Community Edition](https://www.jetbrains.com/pycharm/download/download-thanks.html?platform=windows&code=PCC) - recommended for coding, possible to enable intellisense(limited) for Blender code.
+- [Visual Studio 2019 Community](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=16) - has a powerful python extension, possible to enable intellisense for Blender, provides remote debugging in Blender.
+
+### Coding Conventions
+Aim is to conform to [pep8](https://www.python.org/dev/peps/pep-0008/). 
+At minimum it's 4 spaces for indentation, sources are utf-8, there's `.gitconfig` in the root of the project - please set you editor to use it (for most simplicity). PyCharm default setting are fine and seems that it also picks up `.editorconfig` automatically also, [Tortoise](https://tortoisegit.org/) Merge has a checkbox 'Enable EditorConfig', for Visual Studio there's [EditorConfig extension](https://visualstudiogallery.msdn.microsoft.com/c8bccfe2-650c-4b42-bc5c-845e21f96328).
+
+### Git
+We try to avoid merge commits, the easiest way to do it. This one rejects merges that would result in merge commit:
+```commandline
+> git config [--global] merge.ff only
+```
+Converts pull to do, essentially, fetch&rebase:
+```commandline
+> git config [--global] pull.rebase true
+```
+Also, make more meaningful commits (one commit per feature) the easy way. This will create a single change set from multiple commits coming from `<branch>`:
+```commandline
+> git merge <branch> --squash 
+```
+
+### ThirdParty libraries
+There is ThirdParty repositories included to the project as a submodules. Please update submodules:
+- `deps/MaterialX` https://github.com/AcademySoftwareFoundation/MaterialX
+- `deps/RadeonProRenderUSD` https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD
+- `deps/USD` https://github.com/PixarAnimationStudios/OpenUSD
+
+All of them are included via SSH protocol. You will need to create and install [SSH keys](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh).
+
+Once SSH keys are installed update/checkout submodules for active branch:
+```commandline
+> git submodule update --init -f --recursive
+```
+
+### Build
+Require `python 3.10+` to be set by default.
+
+Download Blender precompiled libraries.
+
+#### Windows:
+Use Open x64 Native Tools Command Prompt for Visual Studio 2019 Community and run.
+```commandline
+> git clone https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon
+> cd BlenderUSDHydraAddon
+> git submodule update --init --recursive
+> python build.py -all -bl-libs-dir <libs_dir> -bin-dir bin -addon

```

Comment:
`-addon` isn't needed with `-all` flag

Reply:
Mention where user can download <libs_dir> without loading blender git. It is possible by
`svn checkout https://svn.blender.org/svnroot/bf-blender/trunk/lib/win64_vc15` for Windows